tests/phpunit/Thelia/Tests/Api/TaxRuleControllerTest.php

```diff
@@ -230,12 +226,10 @@ public function testUpdateTaxRule($taxRuleId)
 
     /**
      * @param $taxRuleId
-     * @depends testCreateTaxRule
+     * @depends testUpdateTaxRule
```

Comment:
To explicit specify create->update->delete test flow.
Actually that does not change anything coz delete test follow after update test, but explicit better than implicit :smile: 
And I think that was just copy/paste issue (`testCreateTaxRule` instead of `testUpdateTaxRule`)